waku/v2/node/config.nim

```diff
@@ -247,6 +247,21 @@ type
       desc: "WebSocket listening port."
       defaultValue: 8000
       name: "websocket-port" }: Port
+    
+    websocketSecureSupport* {.
+      desc: "Enable secure websocket:  true|false",
+      defaultValue: false
+      name: "websocket-secure-support"}: bool
+    
+    websocketSecureKey* {.
+      desc: "Secure websocket key path:   '/path/to/key.txt' ",
+      defaultValue: "key.txt"
```

Comment:
Having a default value here seems insecure. If we don't support auto generating it, I'd prefer if we we treated websocketSecureSupport=true with no key/cert as an error.